style.css

```diff
@@ -53,20 +53,62 @@ a {
   cursor: pointer;
 }
 
+.menu.menu--close {
+  display: none;
+}
+
+.bi-x-lg {
+  display: none;
+}
+
+.bi-x-lg.xopen {
+  display: block;
+  font-size: 20px;
+  color: white;
+  z-index: 999;
+  cursor: pointer;
+  font-weight: bold;
+}
+
 .nav-list {
   display: none;
-  padding: 5px 3px;
+}
+
+.nav-list.nav--open {
+  display: flex;
+  flex-direction: column;
+  list-style-type: none;
+  background-color: #6070ffc2;
+  backdrop-filter: blur(3px);
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  padding: 100px 0 0 30px;
+  margin: 0;
 }
 
 .nav-list .nav__item a {
   border: 3px solid transparent;
   transition: all 0.5s linear;
 }
 
+.nav-list.nav--open .nav__item a {

```

Comment:
- [ ] Kindly make sure to replace the correct background colour for the menu on line 81 with #6070ff.
- [ ] Kindly make sure that when the menu is displayed, the outside background of the page is blurred as per the design.
- [ ] Kindly follow the spacing between mobile menu options as shown in the video.

https://user-images.githubusercontent.com/70269507/173540660-56a34eb9-d300-49ca-93fa-38af90d23067.mov

Reply:
**thanks, I fixed all your requirements.**